lib/screens/lobby.dart

```diff
@@ -20,14 +24,108 @@ class LobbyScreen extends StatelessWidget {
+            stream: context.watch<DatabaseService>().streamOfCreatedGames,
+            builder: (context, snapshot) {
+              if (snapshot.data == null) {
+                print('still null');
+                return Container();
+              } else {
+                print(' stream builder working');
+                return Padding(
+                  padding: const EdgeInsets.fromLTRB(0, 10, 0, 0),
+                  child: ListView(
+                    children: <Widget>[
+                      for (var game in snapshot.data) GameTile(game)
+                    ],
+                  ),
+                );
+              }
+            }),
+      ),
+    );
+  }
+}
+
+class GameTile extends StatelessWidget {
+  final Game _game;
+  GameTile(this._game);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      key: Key('created_game_tile_${_game.id}'),
+      child: Padding(
+        padding: EdgeInsets.fromLTRB(20, 5, 20, 5),
+        child: Card(
+          elevation: 2,
+          child: Padding(
+            padding: EdgeInsets.all(20),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Padding(
+                  padding: EdgeInsets.fromLTRB(0, 0, 100, 0),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Text(
+                        '500m away', // will replace with real data in ticket #60
+                        textAlign: TextAlign.left,
+                        style: TextStyle(
+                          fontSize: 14,
+                          color: Colors.black.withOpacity(0.6),
+                        ),
+                      ),
+                      Text(
+                        _game.name, // title of game
+                        textAlign: TextAlign.left,
+                        style: TextStyle(
+                          fontSize: 24,
+                        ),
+                      ),
+                      Text(
+                        'created by ${_game.creatorName}',
+                        textAlign: TextAlign.left,
+                        style: TextStyle(
+                          fontSize: 14,
+                          color: Colors.black.withOpacity(0.6),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                OutlineButton(
+                  key: Key('join_created_game_tile__btn_${_game.id}'),
+                  textColor: Theme.of(context).primaryColor,
+                  borderSide: BorderSide(color: Theme.of(context).primaryColor),
+                  child: Text('Join'),
+                  onPressed: () {
+                    // navigate to input password screen
+                    Navigator.push<dynamic>(
```

Comment:
Might be a good idea doing all your navigation through the navigation service - probably doesn't make any difference now but having it all in one place may be be an advantage later if you want to do something whenever you navigate for example.